### PR TITLE
Add system-metrics README

### PR DESCRIPTION
**Pull request by ivanayov**
Describe what `system-metrics` function does, how to be invoked
together with some notes for function usage

Signed-off-by: Ivana Yovcheva <iyovcheva@vmware.com>

Follows up #182

Refers to issue #162

### CODE REVIEW

File: system-metrics/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+It queries Prometheus for a function's 2xx and non-2xx invocations count and returns a JSON response of type:
+
+```json
+{
+    "success": 10,
+    "failure": 8
+}
+```
+
+It takes function's name (f.e. `myFunction`) from a query and is invoked by GET request to
```
Comment (alexellis) [inline system-metrics/README.md]: f.e - is this meant to be i.e.?

File: system-metrics/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+http://gateway-url:8080/function/system-metrics?function=myFunction
```
Comment (alexellis) [inline system-metrics/README.md]: Does it only work on query-string or HTTP path also?

Comment (ivanayov) [reply system-metrics/README.md]: For the moment it doesn't work with a path. Does this need an update?

Comment (alexellis) [reply system-metrics/README.md]: I think it's ok for now

**Pull request merged**